node-tests/blueprints/controller-test.js

```diff
@@ -118,32 +86,19 @@ describe('Blueprint: controller', function () {
 
       it('controller foo.js --pod podModulePrefix', function () {
         return emberGenerateDestroy(['controller', 'foo.js', '--pod'], (_file) => {
-          expect(_file('app/pods/foo.js/controller.js')).to.not.exist;
-          expect(_file('tests/unit/pods/foo.js/controller-test.js')).to.not.exist;
```

Comment:
Why remove these?

Reply:
🤔 looking at this diff, it seems like there were some copy/paste shenanigans going on with the pod tests. gonna take another pass on this file